### DISCUSSION TITLE
Can MA be installed manually without HACS?

### DISCUSSION THREAD

**Discussion by rarroyo6** [music-assistant/support#960] [Q&A] [answered]
Are there any instructions for installing Music Assistant manually without using HACS?

**Comment by OzGav** [comment, marked as answer]
No there are multiple repos to install so a bit complex for most

**Reply by Altycoder** [reply]
Please create a custom component for all us non-HACS users out there

**Reply by OzGav** [reply]
The integration is being rolled into HA core in seven days.

**Reply by Altycoder** [reply]
Ah wonderful, hadn't realised that, thanks.  HA 2024.12.x I assume?

**Comment by OzGav** [comment]
Relates to v1
